web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -165,7 +165,7 @@ const MemberSelectorDialog = ({
         );
       }
     }
-  }, [open]);
+  }, [open, selectedMembers]);
```

Comment:
The `initializeChecked` function called in this `useEffect` uses the value of  `selectedMembers`.